app/invocation/invocation_cache_card.tsx

```diff
@@ -12,21 +12,28 @@ const BITS_PER_BYTE = 8;
 
 export default class CacheCardComponent extends React.Component<Props> {
   render() {
+    const hasCacheStats =
+      this.props.model.cacheStats.length &&
+      (+this.props.model.cacheStats[0]?.actionCacheHits != 0 ||
+        +this.props.model.cacheStats[0]?.actionCacheMisses != 0 ||
+        +this.props.model.cacheStats[0]?.totalDownloadSizeBytes != 0 ||
+        +this.props.model.cacheStats[0]?.totalUploadSizeBytes != 0);
```

Comment:
nit: prefer strict equality `!==` to loose equality `!=`

Reply:
Done